### DISCUSSION TITLE
Processing CSSStyleDeclaration'esq css

### DISCUSSION THREAD

**Discussion by Juice10** [postcss/postcss#1616] [Q&A] [answered]
I'm trying to process CSS supplied as json, eg. `{backgroundImage: 'url(./image.png)', color: 'yellow'}`.

```js
const cssSyntax = {backgroundImage: 'url(./image.png)', color: 'yellow'};
const result = await postcss(plugins).process(cssSyntax);
```

This doesn't work as-is, does anyone know how I could do this?

**Comment by ai** [comment, marked as answer]
You need to use this parser https://github.com/postcss/postcss-js/

But your CSS must have selector, not just property list.

**Reply by Juice10** [reply]
Thanks @ai! 
I can wrap it in a selector, and afterwards unwrap it, and postcss-js should totally work for my use case, thanks!
